packages/cubejs-schema-compiler/src/compiler/CubeValidator.js

```diff
@@ -470,7 +470,7 @@ const baseSchema = {
   public: Joi.boolean().strict(),
   joins: Joi.object().pattern(identifierRegex, Joi.object().keys({
     sql: Joi.func().required(),
-    relationship: Joi.any().valid('hasMany', 'belongsTo', 'hasOne').required()
+    relationship: Joi.any().valid('belongsTo', 'belongs_to', 'many_to_one', 'hasMany', 'has_many', 'one_to_many', 'hasOne', 'has_one', 'one_to_one').required()
```

Comment:
We should support new relationship names in camel case too.

Reply:
Ok, no problem 👍